dts/bindings/auxdisplay/sparkfun,serlcd.yaml

```diff
@@ -32,3 +34,23 @@ properties:
     enum:
       - 2
       - 4
+
+  command-delay:
+    type: int
+    default: 10
+    description: |
+      Delay in milliseconds after a normal command was sent.
+      The default value is based on the original SparkFun SerLCD library
+      implementation which assumes 100 kbps I2C configuration. This value
+      might require tweaking if using I2C at a higher bitrare and/or relativily
+      high update frequency of the display.
```

Comment:
default values should be mentioned in the binding description

Reply:
fixed?

Reply:
yup, thanks